tests/integration/nm/ovs_test.py

```diff
@@ -149,13 +147,23 @@ def _attach_port_to_bridge(port_state):
     port_profile_name = nm.ovs.PORT_PROFILE_PREFIX + port_state[OB.PORT_NAME]
 
     _create_proxy_port(port_profile_name, port_state)
-    if port_state[OB.PORT_TYPE] == OBPortType.SYSTEM:
+    if _is_ethernet(port_state[OB.PORT_NAME]):
         _connect_interface(port_profile_name, port_state)
-    elif port_state[OB.PORT_TYPE] == OBPortType.INTERNAL:
+    else:
         iface_name = port_state[OB.PORT_NAME]
         _create_internal_interface(iface_name, master_name=port_profile_name)
 
 
+def _is_ethernet(iface_name):
```

Comment:
Won't it be more correct to name it `_is_system_interface`?

Reply:
Done